src/main/java/com/r307/arbitrader/service/TradingScheduler.java

```diff
@@ -0,0 +1,232 @@
+                try {
+                    activePosition = objectMapper.readValue(stateFile, ActivePosition.class);
+
+                    LOGGER.info("Loaded active trades from file: {}", stateFile.getAbsolutePath());
+                    LOGGER.info("Active trades: {}", activePosition);
+                } catch (IOException e) {
+                    LOGGER.error("Unable to parse state file {}: ", stateFile.getAbsolutePath(), e);
+                }
+            }
+        }
+    }
+
+
+
+    /**
+     * As often as once per minute, display a summary of any non-critical error messages. Summarizing them greatly
+     * reduces how noisy the logs are while still providing the same information.
+     */
+    @Scheduled(cron = "0 * * * * *")
+    public void errorSummary() {
+        if (!errorCollectorService.isEmpty()) {
+            errorCollectorService.report().forEach(LOGGER::info);
+            errorCollectorService.clear();
+        }
+    }
+
+    /**
+     * Display a summary once every 6 hours with the current spreads.
+     */
+    @Scheduled(cron = "0 0 0/6 * * *") // every 6 hours
+    public void summary() {
+        LOGGER.info("Summary: [Long/Short Exchanges] [Pair] [Current Spread] -> [{} Spread Target]", (activePosition != null ? "Exit" : "Entry"));
+
+        List<TradeCombination> tradeCombinations = tickerService.getTradeCombinations(false);
+
+        tradeCombinations.forEach(tradeCombination -> {
+            Spread spread = spreadService.computeSpread(tradeCombination);
+
+            if (spread == null) {
+                return;
+            }
+
+            if (activePosition == null && BigDecimal.ZERO.compareTo(spread.getIn()) < 0) {
+                LOGGER.info("{}/{} {} {} -> {}",
+                    spread.getLongExchange().getExchangeSpecification().getExchangeName(),
+                    spread.getShortExchange().getExchangeSpecification().getExchangeName(),
+                    spread.getCurrencyPair(),
+                    spread.getIn(),
+                    tradingConfiguration.getEntrySpread());
+            } else if (activePosition != null
+                && activePosition.getCurrencyPair().equals(spread.getCurrencyPair())
+                && activePosition.getLongTrade().getExchange().equals(spread.getLongExchange().getExchangeSpecification().getExchangeName())
+                && activePosition.getShortTrade().getExchange().equals(spread.getShortExchange().getExchangeSpecification().getExchangeName())) {
+
+                LOGGER.info("{}/{} {} {} -> {}",
+                    spread.getLongExchange().getExchangeSpecification().getExchangeName(),
+                    spread.getShortExchange().getExchangeSpecification().getExchangeName(),
+                    spread.getCurrencyPair(),
+                    spread.getOut(),
+                    activePosition.getExitTarget());
+            }
+        });
+    }
+
+    @Scheduled(initialDelay = 5000, fixedRate = 3000)
+    public void tick() {
```

Comment:
This has become just a way to drive polling for price data. Let's rename it from `tick()` to `pollForPriceData()` or something like that. It started out as the main event loop so it needed a more generic name.